docs/REST API/rest-api.md

```diff
@@ -2601,18 +2601,17 @@ Returns the information of the specified host.
 
 ### Get host by identifier
 
-Returns the information of the host specified using the `uuid`, `hardware_serial`, `osquery_host_id`, `hostname`, or
-`node_key` as an identifier.
+Returns the information of the host specified using the `hostname`, `uuid`, or `hardware_serial` as an identifier.
```

Comment:
@marko-lisica is this proposed change just for the docs, or should we actually update this API to only accept these 3 types of host identifier, which would be a breaking change (and would increase the scope of this ticket)?

Reply:
@jacobshandling It's just a change for the docs. We don't want to break the API. We want to remove `node_key` and `osquery_host_id` from docs since there's no way to get this in Fleet.

Reply:
cc @mostlikelee

Reply:
it's possible this may lead to confusion/bugs in the case where Device A's osquery_host_id is the same as Device B's hostname.

Reply:
@mostlikelee How usually `osquery_host_id` looks like? AFAIK, there's no way to get it in Fleet?

Reply:
Hey @marko-lisica, yes please look into what `osquery_host_id` looks like so we can understand.

If I'm understanding correctly, @mostlikelee we might be using `osquery_host_id` in places in the code? Is that right?

If that's the case then maybe we leave it in the docs for now so contributors are aware.

Reply:
UDPATE: I think let's merge in the doc changes w/ `osquery_host_id` removal. It sounds like neither contributors nor users use it.

We won't remove it from the code.

If we're wrong, we can add it back to the docs.

Please let me know if folks have any concerns w/ this.